blog/2025-06-05.md

```diff
@@ -0,0 +1,75 @@
+authors: mayya
+tags: [tanager, release]
+---
+
+We’re excited to announce the **0.5 release** milestone for the [Ed-Fi Data
+Management Service
+(DMS)](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/), marking
+another step forward on our path toward the upcoming **Release Candidate 1**.
+This release delivers key features to support authorization, extensibility, and
+interoperability.
+
+<!-- truncate -->
+
+## Key Updates
+
+1. **MetaEd-Based Extensions**: Adds support for TPDM and custom extensions
+   authored using MetaEd.
+2. **SDK Generation**: Automates client SDK generation to facilitate client
+   application development.
+3. **Dependency Endpoint & XSD API**: Supports XML bulk load testing and
+   compatibility validation.
+4. **Claimset-Based Authorization Metadata**: DMS Configuration Service* now
+   includes claimset-based authorization metadata endpoint to facilitate request
+   authorization in Data Management Service.
+5. **Relationship-Based Authorization**: Enables access control via [student
+   enrollment](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/AUTHORIZATION/DOCUMENT-AUTHORIZATION/STUDENT-EDORG-AUTHORIZATION.md)
+   and [staff
+   employment](https://github.com/Ed-Fi-Alliance-OSS/Project-Tanager/blob/main/docs/DMS/AUTHORIZATION/DOCUMENT-AUTHORIZATION/STAFF-EDORG-AUTHORIZATION.md).
+
+*Note: DMS Configuration Service is a separate component designed to manage
+claimsets, authorization metadata, and configuration data for DMS deployments.
+
+📌[View Full
+Changelog](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/compare/v0.4.0...v0.5.0)
+
+## Try It Out
+
+With Docker Desktop or Podman installed locally:
+
+```powershell
+git clone https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service
+cd Data-Management-Service
+git checkout v0.5.0
+
+# Start all required services inside of Docker
+cd eng\docker-compose
+cp .env.example .env
+.\start-local-dms.ps1 -EnableSearchEngineUI -EnableConfig -r
+```
+
+The `.http` files in `src/dms/tests/RestClient` can help you understand how to
+interact with these systems. In particular, see
+`authorization-demonstration-Dec2024.http`. These files should be executable in
+[Visual
+Studio](https://learn.microsoft.com/en-us/aspnet/core/test/http-files?view=aspnetcore-9.0),
+VS Code via
+[rest-client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
+or
+[httpYac](https://marketplace.visualstudio.com/items?itemName=anweber.vscode-httpyac),
+and
+[Rider](https://www.jetbrains.com/help/rider/Http_client_in__product__code_editor.html).
+
+## Next
+
+The 0.5.0 release includes all planned functionalities for the upcoming
+**Release Candidate 1**, which will cover end-to-end testing as well as
+compatibility testing using the [smoke test
+utility](/reference/ods-api/platform-dev-guide/utilities/smoke-test-utility) and
+[bulk load
+utility](/reference/ods-api/platform-dev-guide/utilities/bulk-load-client-utility).
```

Comment:
This is a little confusing to me: if 0.5 already has the planned functionality, then what else is being done in RC 1? Does this mean that the next milestone is just about doing more detailed testing and bug fixes?